server/MongoDb/db.connect.js

```diff
@@ -0,0 +1,15 @@
+const express=require('express');
+const mongoose=require('mongoose');
+
+const uri="mongodb+srv://ankitdb:Singh%40ni%40190@neog-cluster.fs70b.mongodb.net/inventory?retryWrites=true&w=majority"
```

Comment:
- Generally DB url will have sensitive information like DB passwords, so it should be in a `.env` file.

Reply:
done